### ISSUE TITLE
problem New-SafeguardAccessRequest 

### ISSUE THREAD

**Issue by josema993** [OneIdentity/safeguard-ps#453]
With version 6.13.62645 we are having problems when creating a new access request

![image](https://user-images.githubusercontent.com/28726933/212753602-46f0aeda-5c94-4419-9f73-eb654ffe7cd1.png)

when execute the command:

$Appliance = "appliancename"
$user = "user"
$Password = "password" | ConvertTo-SecureString -AsPlainText -Force
$accountName = "machine"
$accountAsset = "machine.url.global"
$sourceappliance = Connect-Safeguard $Appliance -Insecure -IdentityProvider ad1 -Username $user -Password $Password
New-SafeguardAccessRequest -Appliance $Appliance -insecure -AccessToken $sourceappliance -AccountToUse $accountName -AssetToUse $accountAsset -AccessRequestType "SSH"

We get the following error:

Unable to find a requestable account matching 'machine'
En C:\Users\jrodrige\Documents\WindowsPowerShell\Modules\safeguard-ps\6.13.62645\requests.psm1: 100 Carácter: 13
+             throw "Unable to find a requestable account matching '$Ac ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Unable to find ... 'machine':String) [], RuntimeException
    + FullyQualifiedErrorId : Unable to find a requestable account matching 'machine'

Can you help whit us problem?

**Comment by miroberts** [comment]
I believe you are hitting the same issue that is reported here
https://github.com/OneIdentity/safeguard-ps/issues/452 
Does that sound right?

**Comment by josema993** [comment]
> I believe you are hitting the same issue that is reported here #452 Does that sound right?

No. We use version 6.13.62645. And our problem is FullyQualifiedErrorId : Unable to find a requestable account matching 'machine'

Thanks :S

**Comment by miroberts** [comment]
True. I will look into this to see what I can find.

**Comment by miroberts** [comment]
What version of Safeguard are you running against? Is it version 6.13?

**Comment by josema993** [comment]
> What version of Safeguard are you running against? Is it version 6.13?

We use 
![image](https://user-images.githubusercontent.com/28726933/213107492-37e90749-2edb-4298-9e2e-cd56e3030751.png)

and the safeguard-ps version is: 6.13.62645

**Comment by miroberts** [comment]
I have tested with safeguard-ps 6.13.6245 / spp 7.1 and can't duplicate this. Are you able to checkout the password for account "machine" from the UI?

**Comment by MattBeaudinQC** [comment]
Hi @josema993 ,

I'm going to close this one out. If this is still an issue, please let us know,

Matt
